numba_dpex/kernel_api_impl/spirv/target.py

```diff
@@ -220,6 +218,75 @@ def _generate_spir_kernel_wrapper(self, func, argtypes):
         module.get_function(func.name).linkage = "internal"
         return wrapper
 
+    def _populate_array(
```

Comment:
I would prefer to be it consistent with numba implementation and implemented as a function in arrayobj.py